pkg/lib/authenticationflow/declarative/data_create_identity.go

```diff
@@ -0,0 +1,41 @@
+package declarative
```

Comment:
The filename `data_` makes me think the structs in this file will appear in the HTTP API `data`. But it seems not. Maybe move these structs to `utils_account_linking.go`?